WebDriverAgentLib/Categories/XCUIElement+FBUtilities.m

```diff
@@ -255,8 +288,8 @@ - (NSData *)fb_screenshotWithError:(NSError **)error
           || (appFrame.size.width > appFrame.size.height && parentWindowFrame.size.width > parentWindowFrame.size.height)
           || (appFrame.size.width < appFrame.size.height && parentWindowFrame.size.width < parentWindowFrame.size.height)) {
         CGPoint fixedOrigin = orientation == UIInterfaceOrientationLandscapeLeft ?
-          CGPointMake(appFrame.size.height - elementRect.origin.y - elementRect.size.height, elementRect.origin.x) :
```

Comment:
I think it makes sense to keep the previous indentation

Reply:
I can revert this part but this was auto indentation from XCode. Just FYI